doc/man3/CMS_decrypt.pod

```diff
@@ -82,6 +95,9 @@ The error can be obtained from ERR_get_error(3).
 
 =head1 BUGS
 
+The B<set1_> part of these function names is misleading
+and should bettter read: B<with_>.
```

Comment:
Typo: better

Reply:
Yeah, `better`  is better :wink: - fixed.